pkg/minikube/bootstrapper/certs.go

```diff
@@ -638,6 +638,55 @@ func isKubeadmCertValid(cmd command.Runner, certPath string) bool {
 	return err == nil
 }
 
+// EnsureCACertsEarly collects host-provided custom CA certs, copies them into the guest,
+// installs symlinks into the system trust store, and refreshes trust *before* HTTPS probes.
+func EnsureCACertsEarly(cr command.Runner) error {
+	caCerts, err := collectCACerts()
```

Comment:
the collectCACerts is also called in `func SetupCerts` 
would that make it duplicate or do two times? I would like to ensure this wont make minikube start slower, can it be done once for every minikbube start?